core/tests/test_utils.py

```diff
@@ -71,11 +72,28 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Sets the content, status code and headers that the urlfetch mock
+        should return with.
+
+        Args:
+            content: str. The content to return in subsequent calls to the
+            urlfetch mock.
+            status_code: int. The status_code to return in subsequent calls
+            to the urlfetch mock.
+            headers: dict. The headers to return in subsequent calls to
```

Comment:
I agree that, in this case, there is no need to outline the specific keys. However, think about how this is used -- it's used when a test wants to specify that a urlfetch request should execute with specific headers. So, the docstring here should also say something like "The keys of the dict are strings represent the header name and the corresponding values are strings representing the value of that header." (or something like that).